### ISSUE TITLE
No data source for aws_route53_delegation_set

### ISSUE THREAD

**Issue by rmakram-ims** [terraform-providers/terraform-provider-aws#6073]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

There is no data source for the aws_route53_delegation_set.

### New or Affected Resource(s)

* aws_route53_delegation_set

### Potential Terraform Configuration

```hcl
data "aws_route53_delegation_set" "main" {
  reference_name = "DynDNS"
}

resource "aws_route53_zone" "primary" {
  name              = "hashicorp.com"
  delegation_set_id = "${data.aws_route53_delegation_set.main.id}"
}

```

### References

* #0000

**Comment by bflad** [comment]
A new `aws_route53_delegation_set` data source has been merged and will release with version 1.47.0 of the AWS provider, sometime this week.

**Comment by bflad** [comment]
This was released (4 days ago) in [version 1.47.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1470-november-26-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
